src/Mod/Fem/femtools/femutils.py

```diff
@@ -120,7 +120,7 @@ def is_derived_from(obj, t):
 def get_pref_working_dir(solver_obj):
     """ Return working directory for solver honoring user settings.
 
-    :throws femtools.erros.MustSaveError:
+    :throws femtools.errors.MustSaveError:
```

Comment:
@berndhahnebach this should fix something :stuck_out_tongue_winking_eye:

Reply:
thanks for thoroughly investigating that and also _killing my buzz_ :stuck_out_tongue: